virttest/env_process.py

```diff
@@ -1123,16 +1146,33 @@ def preprocess(test, params, env):
                 testlist = [new_params.get("avocado_guest_vt_test",
                                            "boot")]
                 avocadotestargs = new_params.get("avocado_guest_add_args", "")
+                # add nested-hv feature attribute to VM's libvirt xml
+                if new_params.get("vm_type") == "libvirt":
```

Comment:
we can choose to use --virt-install extra-params aswell, if create_vm_libvirt param is set, so that we can avoid creating a vm at all to change.

Reply:
Thanks @sathnaga, I have taken care of the suggestion.